### PR TITLE
added a call us tag

### LINKED ISSUE #208: [Feature] add AE phone number in corrective action

### PR DESCRIPTION
**Pull request by JavaLavaMT**
closes #208

### CODE REVIEW

File: app/templates/account/employee/index/counseling/issue/new.hbs [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <hgroup>
+  <p style="text-align:right">If you need assistance call: <span style="font-weight:bold"><a href="tel:(406)248-6178">(406) 248-6178</a></span></p>
```
Comment (James1x0) [inline app/templates/account/employee/index/counseling/issue/new.hbs]: Don't use style attributes to style elements, locate them into a less file.

**Comment by JavaLavaMT** [comment]
screen shot 2018-07-03 at 7 53 12 am

**Comment by James1x0** [comment]
I'm _ok_ with the style here. I want it changed in the future (but alongside some support integration). For now this satisfies the requirements.

File: app/templates/account/employee/index/counseling/issue/new.hbs [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <hgroup>
+  <p class="right">If you need assistance call: <span><a href="tel:(406)248-6178">(406) 248-6178</a></span></p>
```
Comment (James1x0) [inline app/templates/account/employee/index/counseling/issue/new.hbs]: Use classnames `right aligned text` and use `strong` tags for bolding. No need to make extra styles here.

**Pull request merged**